superset/cli.py

```diff
@@ -212,180 +221,305 @@ def refresh_druid(datasource: str, merge: bool) -> None:
-    from superset.dashboards.commands.importers.dispatcher import (
-        ImportDashboardsCommand,
-    )
-
-    path_object = Path(path)
-    files: List[Path] = []
-    if path_object.is_file():
-        files.append(path_object)
-    elif path_object.exists() and not recursive:
-        files.extend(path_object.glob("*.json"))
-    elif path_object.exists() and recursive:
-        files.extend(path_object.rglob("*.json"))
-    if username is not None:
-        g.user = security_manager.find_user(username=username)
-    contents = {path.name: open(path).read() for path in files}
-    try:
-        ImportDashboardsCommand(contents).run()
-    except Exception:  # pylint: disable=broad-except
-        logger.exception("Error when importing dashboard")
+if feature_flags.get("VERSIONED_EXPORT"):
 
+    @superset.command()
+    @with_appcontext
+    @click.option(
+        "--dashboard-file",
+        "-f",
+        default="dashboard_export_YYYYMMDDTHHMMSS",
+        help="Specify the the file to export to",
+    )
+    def export_dashboards(dashboard_file: Optional[str]) -> None:
+        """Export dashboards to ZIP file"""
+        from superset.dashboards.commands.export import ExportDashboardsCommand
+        from superset.models.dashboard import Dashboard
+
+        g.user = security_manager.find_user(username="admin")
+
+        dashboard_ids = [id_ for (id_,) in db.session.query(Dashboard.id).all()]
+        timestamp = datetime.now().strftime("%Y%m%dT%H%M%S")
+        root = f"dashboard_export_{timestamp}"
+        dashboard_file = dashboard_file or f"{root}.zip"
+
+        with ZipFile(dashboard_file, "w") as bundle:
+            for file_name, file_content in ExportDashboardsCommand(dashboard_ids).run():
+                with bundle.open(f"{root}/{file_name}", "w") as fp:
+                    fp.write(file_content.encode())
+
+    # pylint: disable=too-many-locals
+    @superset.command()
+    @with_appcontext
+    @click.option(
+        "--datasource-file",
+        "-f",
+        default="dataset_export_YYYYMMDDTHHMMSS",
+        help="Specify the the file to export to",
+    )
+    def export_datasources(datasource_file: Optional[str]) -> None:
+        """Export datasources to ZIP file"""
+        from superset.connectors.sqla.models import SqlaTable
+        from superset.datasets.commands.export import ExportDatasetsCommand
+
+        g.user = security_manager.find_user(username="admin")
+
+        dataset_ids = [id_ for (id_,) in db.session.query(SqlaTable.id).all()]
+        timestamp = datetime.now().strftime("%Y%m%dT%H%M%S")
+        root = f"dataset_export_{timestamp}"
+        datasource_file = datasource_file or f"{root}.zip"
+
+        with ZipFile(datasource_file, "w") as bundle:
+            for file_name, file_content in ExportDatasetsCommand(dataset_ids).run():
+                with bundle.open(f"{root}/{file_name}", "w") as fp:
+                    fp.write(file_content.encode())
```

Comment:
We could try except and surface a prettier message to the user's in case somethings goes wrong